pkg/http-server/start.go

```diff
@@ -54,14 +56,20 @@ func (g *APIServer) start(routes []*Route, port, certFile, privateKeyFile string
 		router = mux.NewRouter()            // new router
 	)
 
+	logWriter := logging.GetLogWriter(logger)
```

Comment:
I am wondering, why do we need a separate logwriter? The already  created `logger` won't help in this case?

Reply:
the logwriter consumes the logger.

logger object doesn't implement the Writer interface. so had to create a logwriter object that does.